pyfr/integrators/std/controllers.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 
 import math
+import numpy as np
```

Comment:
Add newline.  (NumPy is not a standard Python package and so goes into its own stanza.)